lib/kino/data_table.ex

```diff
@@ -242,6 +242,15 @@ defmodule Kino.DataTable do
 
   defp value_to_string(value) when is_atom(value), do: inspect(value)
 
+  defp value_to_string(value) when is_list(value) do
+    try do
+      List.to_string(value)
+    rescue
+      ArgumentError ->
+        Enum.map(value, &value_to_string/1) |> inspect()
```

Comment:
Do we need value_to_string if we are going to inspect it anyway?

Reply:
+1, if we inspect we should be consistent with a regular inspect :)

Reply:
Not at all, my tired brain thought so... 🙄